python/doc/examples/data_analysis/distribution_fitting/plot_estimate_gev_fremantle.py

```diff
@@ -0,0 +1,166 @@
+"""
+Estimate a GEV on the Fremantle sea levels data
+===============================================
+"""
+# %%
+# In this example we are going to estimate the parameters of a generalized extreme value distribution
+# on sea levels data from [coles2001]_ with different methods:
+#
+# - With the maximum likelihood method
+# - With the profile likelihood method
+# - With the estimation of return level from maximum likelihood and profile likelihood
+# - With the maximum the log-likehood estimator from independent observations (see [coles2001]_ paragraph 3.4.1)
+# - With the maximum the log-likehood estimator with a non-stationary hypothesis (see [coles2001]_ paragraph 6.3.1)
+
+# %%
+# Load the Fremantle dataset of highest sea levels per year
+import openturns as ot
+import openturns.viewer as otv
+import openturns.experimental as otexp
+from openturns.usecases import coles
+data = coles.Coles().fremantle
+print(data[:5])
+
+# %%
+# Select the sea levels column
+sample = data[:, 1]
+
+# %%
+# Estimate the parameters of the GEV by maximizing the log-likehood
+# and compute the parameter distribution
+factory = ot.GeneralizedExtremeValueFactory()
+result1 = factory.buildMethodOfLikelihoodMaximizationEstimator(sample)
+
+# %%
+# Print the estimated parameter values
+estimate = result1.getDistribution()
+desc = estimate.getParameterDescription()
+p = estimate.getParameter()
+print(", ".join([f"{param}: {value:.3f}" for param, value in zip(desc, p)]))
+
+# %%
+# Inspect the estimated Gaussian parameter distribution
+parameterEstimate = result1.getParameterDistribution()
+print(parameterEstimate)
+
+# %%
+# Inspect the covariance matrix
+V = parameterEstimate.getCovariance()
+print(V)
+
+# %%
+# Inspect the standard deviation
+stddev = parameterEstimate.getStandardDeviation()
+print(stddev)
+
+# %%
+# Extract parameters confidence intervals
+for i in range(3):
+    ci = parameterEstimate.getMarginal(i).computeBilateralConfidenceInterval(0.95)
+    print(desc[i] + ":", ci)
+
+# %%
+# Validate the inference result thanks to some diagnostic plots
+validation = otexp.GeneralizedExtremeValueValidation(result1, sample)
+graph = validation.drawDiagnosticPlot()
```

Comment:
I think each of these four graphs is insteresting in its own right. Could you perhaps create separate draw methods for each (and keep the genereal drawDiagnosticPlot, of course)?

Reply:
This is not critical, and you don't have time to do it before the RC, but it would be nice to have in 1.22.

Reply:
done